hack/observability/grafana/dashboards/runtime-extensions.json

```diff
@@ -18,7 +18,6 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
-  "id": 6,
```

Comment:
Should this be kept? (the other dashboards have an id set)

Reply:
I edited the dashboard in the UX and then saved the json from there, not sure why the ID was dropped

Reply:
As long as the dashboard still works on `tilt up` I'm fine either way. 

In general it would be good if we keep exporting dashboards the same way going forward so we don't go back on forth on things like this (IIRC there are various ways to export or save dashboards in Grafana)